src/styles/card.scss

```diff
@@ -0,0 +1,86 @@
+.card {
+  width: 200px;
+  box-sizing: border-box;
+  border: 1px solid $white-color;
+  padding-top: 32px;
+
+  &__img {
+    display: flex;
+    width: 160px;
+    height: 134px;
+    margin-inline: 19px;
+  }
+
+  &__content {
+    padding: 40px 16px 16px;
+  }
+
+  &__title {
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    color: $main-color;
+    margin-bottom: 4px;
+  }
+
+  &__code {
+    color: $secondary-color;
+    margin-bottom: 16px;
+  }
+
+  &__review {
+    display: flex;
+    justify-content: space-between;
+    margin-bottom: 22px;
+
+    &-text {
+      padding-block: 2px;
+    }
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    margin-bottom: 14px;
+
+    &-text {
+      padding-block: 1px;
+      color: $secondary-color;
+      font-size: 12px;
+      line-height: 18px;
+    }
+
+    &-count {
+      color: $main-color;
+      font-size: 16px;
+      line-height: 18px;
+      font-weight: 700;
+    }
+  }
+
+  &__button {
+    width: 100%;
+    text-align: center;
+
+    &-link {
+      display: block;
+      padding-block: 12px;
+      text-decoration: none;
+      text-transform: uppercase;
+
+      background-color: $button-color;
+      border: 1px solid $button-color;
+      border-radius: 5px;
+
+      font-size: 14px;
+      font-weight: 700;
+      line-height: 16px;
+      color: #fff;
+
+      &:hover {
+        color: #00acdc;
```

Comment:
Again, color values must be stored in variables. Please, define a variable for the color `#00acdc` and use it instead.